app/components/intermission.jsx

```diff
@@ -0,0 +1,196 @@
+// https://github.com/EnCiv/civil-pursuit/issues/137
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import { PrimaryButton, SecondaryButton } from './button'
+import Intermission_Icon from '../svgr/intermission-icon'
+const Intermission = props => {
+  const {
+    className = '',
+    user = {},
+    round = 1, // the round that the user has just completed
+    ...otherProps
```

Comment:
We need to define another prop - lastRound.
If round is less than lastRound then we offer the user the option to Continue.  If round is equal to lastRound (or greater but that shouldn't happen) then we just tell them we will notify them when the next round is available.   It's not based on the round, is based on the test. 

And, we need to add another prop, onDone which defaults to ()=>{}

When the user clicks continue, call onDone({vallid: true, value: 'continue'})
When the user clicks remind me later, call onDone({valid: true, value: 'remind'})

Reply:
Initially I didn't see that you had changed this because I didn't see it in the story:
please make lastRound default to 1, and make it a prop in the story, where it can be set to 2. 

Please add the onDoneDecorator to the stories, see why-step.stories.jsx  so we can see that onDone is getting called.